core/templates/domain/story/story-update.service.spec.ts

```diff
@@ -525,6 +535,92 @@ describe('Story update service', () => {
     }
   );
 
+  it('should set a story node status', () => {
+    expect(
+      _sampleStory.getStoryContents().getNodes()[0].getStatus()
+    ).toBe('Published');
+    storyUpdateService.setStoryNodeStatus(
+      _sampleStory, 'node_1', 'Draft');
+    expect(
+      _sampleStory.getStoryContents().getNodes()[0].getStatus()
+    ).toBe('Draft');
+
+    undoRedoService.undoChange(_sampleStory);
+    expect(
+      _sampleStory.getStoryContents().getNodes()[0].getStatus()
+    ).toBe('Published');
```

Comment:
```suggestion
    undoRedoService.undoChange(_sampleStory);

    expect(
      _sampleStory.getStoryContents().getNodes()[0].getStatus()
    ).toBe('Published');
```

Please do the same for the remaining tests.